index.js

```diff
@@ -397,6 +412,36 @@ async function fastifyStatic (fastify, opts) {
     const routeConfig = req.routeOptions?.config || req.routeConfig
     pumpSendToReply(req, reply, routeConfig.file, routeConfig.rootPath)
   }
+
+  async function generateHashForFiles (rootPath) {
+    fileHashes ||= new Map()
```

Comment:
I think you are looking for `??=`?

Reply:
Yeah, but node 14 won't let me :(

Reply:
Why does v14 matter? CI does not include it.

Reply:
Oh that's not good... v3 should've tested for 14...

Seems like v3.9.2 actually superseded v4